lib/routes/api/metadata/processor.js

```diff
@@ -56,21 +61,30 @@ module.exports = function(url, options) {
       function done(metadata={}) {
         debug('parsed', metadata);
 
-        if (!(metadata.embed || metadata.title)) {
-          return Promise.reject(new Error('empty'));
-        }
-
-        metadata.unadaptedUrl = metadata.id = metadata.url;
-        metadata.displayUrl = decodeURIComponent(metadata.url);
-        metadata.originalUrl = url;
-        metadata.maxAge = result.maxAge;
-
-        const twitterUsername = result.foundParams.magnet_twitter_username;
-        if (twitterUsername) {
-          metadata.twitterUsername = twitterUsername.replace('@', '');
-        }
-
-        return metadata;
+        return contentServiceRequest.then((res) => {
+          return Object.assign(metadata, res.extra_metadata || {});
+        })
+        .catch((err) => {
```

Comment:
Can you add a little comment about when/how a URL could error and why it doesn't matter.